web/web_test.go

```diff
@@ -253,10 +250,7 @@ func TestCheckClientCompatability(t *testing.T) {
 		{"Safari Mobile", "Mozilla/5.0 (iPhone; CPU iPhone OS 9_1 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13B137 Safari/601.1", true},
 	}
 	for _, browser := range uaTestParameters {
-		t.Run(browser.Name, func(t *testing.T) {
-			if result := CheckClientCompatability(browser.UserAgent); result != browser.Result {
-				t.Fatalf("%s User Agent Test failed!", browser.Name)
-			}
-		})
+		result := CheckClientCompatability(browser.UserAgent)
```

Comment:
Can you re-add the `t.Run` around these so that the different test cases still show up as their own subtests?

Reply:
done